pkg/tsdb/influxdb/flux/converters.go

```diff
@@ -0,0 +1,168 @@
+package flux
```

Comment:
It would be nice to have converters as part of an sdk, seems generic enough to move them out eventually. I don't see anything flux specific here, although I realize this is probably a first pass. IMHO